packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -363,12 +363,25 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
       }
 
       const localStream = callState.localVideoStreams.find((item) => item.mediaStreamType === 'Video');
-      if (!localStream) {
-        return;
+      const localScreenSharingStream = callState.localVideoStreams.find(
```

Comment:
What happens in the case that the user turns on their local video and then starts their screen share quickly together? Do we see any error's or weird behavior here? (this should be fine but calling out these questions so that our bases are covered)

Reply:
Tested and didn't see any errors